src/middlewared/setup.py

```diff
@@ -18,22 +18,19 @@
 ]
 
 
-def get_etc_files(*args, **kwargs):
+def get_assets(name):
     """
-    Recursive get dirs from middlewared/etc_files/.
-    This is required for the etc plugin.
+    Recursive get dirs from middlewared/{name}
     """
     base_path = os.path.join(
         os.path.dirname(os.path.realpath(__file__)),
         'middlewared',
-        'etc_files',
     )
-    for root, dirs, files in os.walk(base_path):
-        if base_path == root:
-            yield 'etc_files/*'
-        else:
-            entry = root.replace(base_path, 'etc_files')
-            yield f'{entry}/*'
+    result = []
+    for root, dirs, files in os.walk(os.path.join(base_path, name)):
+        result.append(os.path.relpath(root, base_path))
+        result.append(f'{root}/*')
```

Comment:
@themylogin I am confused why there are two appends here? One with only root which will have absolute paths and another with the relative path?